templates/machine/salvator-xs-h3-4x2g-xt/50_bblayers.conf.inc

```diff
@@ -0,0 +1,6 @@
+BBLAYERS =+ "\
+  ${METADIR}/bsp/meta-renesas-rcar-gen3/meta-rcar-gen3 \
+  ${METADIR}/meta-xt-images-rcar-gen3 \
+  ${METADIR}/meta-xt-agl-base \
+  ${METADIR}/meta-aos \
```

Comment:
Curious question: Do we really need meta-aos here?

Reply:
Removed line